exec/java-exec/src/main/resources/rest/index.ftl

```diff
@@ -192,6 +199,8 @@
       var port = getPortNum();
       var timeout;
       var size = $("#size").html();
+      reloadMemory();
+      setInterval(reloadMemory, refreshTime);
```

Comment:
In graceful shutdown `setTimeout` is used. Which one is better? Anayway it's better to be consistent in scheduler usage. Plus we do why need two schedulers? Can one work for both tasks? Plus if one drillbit is unavailable, there is no need to check memory.

Reply:
That's because the scheduler for graceful shutdown can change, while in this case we need a periodic refresh that shouldn't change.
As for a Drillbit going down, it can always come back online. As long as the page continues to list the Drillbit and it does get updated to reflect the status as "ONLINE" again, we should be pinging for the metrics too.